### ISSUE TITLE
Full text is scrambled through multi-columns

### ISSUE THREAD

**Issue by chlorophyll-zz** [okfde/offenegesetze.de#41]
Full text is scrambled through the multi-columns on pages like this:
https://offenegesetze.de/veroeffentlichung/bgbl1/1974/63#page=1

**Comment by jfilter** [comment]
Yes, that's true. We are looking into it but are not sure if we can (easily) fix it.
